### ISSUE TITLE
Missing libmpfr

### ISSUE THREAD

**Issue by imphil** [lowRISC/lowrisc-toolchains#1]
Running this toolchain (built on Ubuntu 16.04) on my openSUSE Tumbleweed machine gives me:

```
/home/tools/riscv/bin/../libexec/gcc/riscv32-unknown-elf/8.3.0/cc1: error while loading shared libraries: libmpfr.so.4: cannot open shared object file: No such file or directory
```

Looks like the build script isn't good enough :-( Should we look into something like crosstools-ng instead of the SiFive build script @asb?

**Comment by imphil** [comment]
Looking at crosstool-ng now. Seems promising so far.

**Comment by imphil** [comment]
crosstool-ng solves the libmpfr issue and reduces the build time down to 30 minutes. That's a significant win on all fronts, so PR #3 should solve this one.

**Comment by lenary** [comment]
Closed by #3 for ibex.

**Comment by lenary** [comment]
We use crosstool-ng for all builds now, so this is solved.
